drafts/te-types-update/draft-ietf-teas-rfc8776-update.txt

```diff
@@ -8713,1160 +8802,2257 @@ B.1.  TE Types YANG Diffs
 
    The output (model-updates.txt) is reported here:
 
- 21a22,33
- >   import ietf-network {
- >     prefix "nw";
- >     reference
- >       "RFC 8345: A YANG Data Model for Network Topologies";
- >   }
- >
- >   import ietf-network-topology {
- >     prefix "nt";
- >     reference
- >       "RFC 8345: A YANG Data Model for Network Topologies";
- >   }
- >
- 30c42
+ 1,3378c1,4547
+ < module ietf-te-types {
+ <   yang-version 1.1;
+ <   namespace "urn:ietf:params:xml:ns:yang:ietf-te-types";
+ <   prefix te-types;
+ <
+ <   import ietf-inet-types {
+ <     prefix inet;
+ <     reference
+ <       "RFC 6991: Common YANG Data Types";
+ <   }
+ <   import ietf-yang-types {
+ <     prefix yang;
+ <     reference
+ <       "RFC 6991: Common YANG Data Types";
+ <   }
+ <   import ietf-routing-types {
+ <     prefix rt-types;
+ <     reference
+ <       "RFC 8294: Common YANG Data Types for the Routing Area";
+ <   }
+ <
+ <   organization
+ <     "IETF Traffic Engineering Architecture and Signaling (TEAS)
+ <      Working Group";
+ <   contact
+ <     "WG Web:   <https://datatracker.ietf.org/wg/teas/>
+ <      WG List:  <mailto:teas@ietf.org>
+ <
+ <      Editor:   Tarek Saad
  <                <mailto:tsaad@juniper.net>
```

Comment:
Please update my email here

Reply:
This is from a diff with the YANG model in RFC8776 where the juniper mail address was used. The YANG models in RFC8776-bis are all using the gmail address.